.infrastructure/50_storage.tf

```
@@ -22,3 +22,49 @@ resource "aws_s3_bucket_versioning" "cms_medialibrary_bucket" {
     status = "Enabled"
   }
 }
+
+## Infra for Static Site on AWS
+resource "random_integer" "website_bucket_random_integer" {
+  min = 1
+  max = 9999
+}
+
+resource "aws_s3_bucket" "website" {
+  bucket = "website-b2b-${random_integer.website_bucket_random_integer.result}"
+}
+
+resource "aws_s3_bucket_public_access_block" "website" {
+  bucket                  = aws_s3_bucket.website.id
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+resource "aws_s3_bucket_versioning" "website" {
+  bucket = aws_s3_bucket.website.id
+  versioning_configuration {
+    status = "Enabled"
```

What about removing the versioning? IMHO the content into the bucket doesn't required to be versioned

```suggestion
    status = "Disabled"
```

---

i agree